simple_ai_agents/Recruitify/agents.py

```
@@ -0,0 +1,675 @@
+            
+            response = llm.invoke(prompt)
+            skills_text = response.content
+            
+      
+            match = re.search(r'\[(.*?)\]', skills_text, re.DOTALL)
+            if match:
+                skills_text = match.group(0)
+            
+
+            try:
+                skills_list = eval(skills_text)
```

Ruff (0.12.2)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)